release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -515,6 +515,78 @@ For more information, see xref:../security/tls-security-profiles.adoc#tls-profil
 [id="ocp-release-notes-storage_{context}"]
 === Storage
 
+[id="ocp-release-notes-storage-azure-file-cross-sub-support_{context}"]
+==== Azure File cross-subscription support is generally available
+Cross-subscription support allows you to have an {product-title} cluster in one Azure subscription and mount your Azure file share in another Azure subscription using the Azure File Container Storage Interface (CSI) driver.
+
+This feature is generally available in {product-title} 4.19.
+
+// For more information, see ???
+
+[id="ocp-release-notes-storage-vol-attributes_{context}"]
+==== Volume Attributes Classes (Technology Preview)
+Volume Attributes Classes provide a way for administrators to describe "classes" of storage they offer. Different classes might correspond to different quality-of-service levels.
+
+Volume Attributes Classes in {product-title} 4.19 is available only with AWS Elastic Block Storage (EBS) and Google Cloud Platform (GCP) persistent disk (PD) Container Storage Interface (CSI).
+
+You can apply a Volume Attributes Class to a persistent volume claim (PVC). If a new Volume Attributes Class becomes available in the cluster, a user can update the PVC with the new Volume Attributes Class if needed.
+
+Volume Attribute Classes have parameters that describe volumes belonging to them. Different parameters can be accepted depending on the provisioner or the resizer. If a parameter is omitted, the default is used at volume provisioning. If a user applies the PVC with a different Volume Attributes Class with omitted parameters, the default value of the parameters might be used depending on the CSI driver implementation. For more information, see the related CSI driver documentation.
+
+Volume Attributes Classes is available in {product-title} 4.19 with Technology Preview status.
+
+//For more information, see ???.
+
+[id="ocp-release-notes-storage-cli-cmd-pvc-usage_{context}"]
+==== New CLI command to show PVC usage (Technology Preview)
+{product-title} 4.19 introduces a new command to view persistent volume claim usage. This feature has Technology Preview status.
+
+// For more information, see ???.
+
+[id="ocp-release-notes-storage-cli-cmd-resize-recovery_{context}"]
+==== CSI volume resizing recovery is generally available
+Previously, you might expand a persistent volume claim (PVC) to a size that is not supported by the underlying storage provider. In which case, the expansion controller typically tries forever to expand the volume and keeps failing. 
+
+This new feature allows you to recover and provide another resize value for the PVC. Resizing recovery is supported as generally available in {product-title} 4.19.
+
+For more information about resizing volumes, see xref:../storage/expanding-persistent-volumes.adoc[Expanding persistent volumes].
```

Comment:
needs [anchor ID ](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references)